### PR TITLE
Update content type menu to handle reading order for images

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made? 🤔

Fixes https://github.com/sul-dlss/argo/issues/3211

## How was this change tested? 🤨
Automated test suite

### CODE REVIEW

File: app/javascript/controllers/content_type_controller.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    static targets = [ "direction", "directionRow", "contentType"]
+
+    connect() {
+      this.render()
+    }
+
+    render() {
+      switch (this.currentType()) {
+        case 'https://cocina.sul.stanford.edu/models/image':
+        case 'https://cocina.sul.stanford.edu/models/book':
```
Comment (peetucket) [inline app/javascript/controllers/content_type_controller.js:13]: image types can have a reading direction set too?

**Review by peetucket — APPROVED** [review]
Just a question about reading direction for images

File: app/javascript/controllers/content_type_controller.js (hunk 1 above)
Comment (jcoyne) [reply app/javascript/controllers/content_type_controller.js:13]: Yes, that's exactly the change requested in the ticket.

Comment (andrewjbtw) [reply app/javascript/controllers/content_type_controller.js:13]: I was surprised but we found it in existing data and the viewer was parsing it correctly.

**Pull request merged**